### DISCUSSION TITLE
Recommended VSCode Extensions

### DISCUSSION THREAD

**Discussion by MM25Zamanian** [AliMD/alwatr#67] [Ideas]
Please post your ideas for vscode extensions to us: 
@AliMD @asmanp2012 Please comment!

**Comment by MM25Zamanian** [comment]
[runem.lit-plugin](https://marketplace.visualstudio.com/items?itemName=runem.lit-plugin)

**Comment by MM25Zamanian** [comment]
[dbaeumer.vscode-eslint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)

**Comment by MM25Zamanian** [comment]
[stylelint.vscode-stylelint](https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint)

**Comment by alimd** [comment]
My old recomendation fro lit projects please duble check.

```json
{
  "recommendations": [
    "editorconfig.editorconfig",
    "streetsidesoftware.code-spell-checker",
    "runem.lit-plugin",
    "dbaeumer.vscode-eslint",
    "stylelint.vscode-stylelint",
    "christian-kohler.path-intellisense",
    "wayou.vscode-todo-highlight",
    "formulahendry.auto-close-tag",
    "davidanson.vscode-markdownlint",
    "bbugh.change-color-format"
  ]
}
```

**Reply by alimd** [reply]
همش واجب نیست
مثلا `bbugh.change-color-format` با ینکه خیلی مفیده ولی نباید بره در فایل ‍`extentions.json`

**Comment by alimd** [comment]
لطفا روال این باشه که اکستنشن های مفید که پیدا میکنیم رو اینجا بیان کنیم بعد بهش نظر بدید
اونایی که واجب هست میره تو `extentions.json` مابقی هم میمونه برای هرکی دوست داشت استفاده کنه

**Reply by MM25Zamanian** [reply]
خب مگه وقتی میره توی ریکامندد های extensions.json نصبش واجب میشه؟؟

**Reply by alimd** [reply]
نه فقط به کاربر نشون میده
ربط سوالت به حرف منو نفهمیدم
درضمن چرا زودتر نمیسازی و انقدر طول میدی؟
